src/java/org/apache/cassandra/repair/SystemDistributedKeyspace.java

```diff
@@ -74,48 +74,50 @@ private SystemDistributedKeyspace()
 
     private static final TableMetadata RepairHistory =
```

Comment:
Hmm... well primarily because it was extra work and we can drop the columns later. I don't see a  reason not to use the built in schema change capabilities.

Other cases that have a migration and new table do that because the key changed and we don't support changing key columns.

Reply:
ughh, i did realize this later, but I though i had updated the comment. oh well, sgtm